modules/data.atmosphere/R/download.US_Los.R

```diff
@@ -0,0 +1,116 @@
+##' @title download.US-Los
+##' 
+##' @section General Description:
+##' Obtains data from Ankur Desai's Lost Creek flux tower, and selects certain variables (NEE and LE) to return
+##' Data is retruned at the given timestep in the given range.
+##' 
+##' This data includes information on a number of flux variables.
+##' 
+##' The timestep parameter is measured in hours, but is then converted to half hours because the data's timestep
+##' is every half hour.
+##' 
+##' @param start_date Start date/time data should be downloaded for
+##' @param end_date End date/time data should be downloaded for
+##' @param timestep How often to take data points from the file.  Must be a multiple of 0.5
+##' @export
+##' 
+##' @author Luke Dramko and K Zarada
+download.US_Los <- function(start_date, end_date, timestep = 1) {
+  timestep = 2 * timestep #data is actually every half hour
+  
+  if (timestep != as.integer(timestep)) {
+    PEcAn.logger::logger.severe(paste0("Invalid timestep ", timestep/2, ". Timesteps must be at ",
+                                       "least every half hour (timestep = 0.5)."))
+  }
+  
+  start_date <- as.POSIXct(start_date, tz="UTC")
+  end_date <- as.POSIXct(end_date, tz="UTC")
+  
+  nee_col = 33  # Column number of NEE
```

Comment:
can you get this by column name or something instead?

Reply:
@kzarada I think the base_url link in the code is also broken.

Reply:
@para2x I actually don't think this file is necessary. I accidentally changed it in the wrong branch back when we were working on this a couple of months ago.

Reply:
cool cool. I'm gonna delete it then